media/js/facebookapps/App.js

```diff
@@ -101,8 +101,13 @@ DOWNLOADTAB.classes.App = (function (singleton) {
                 self.theater.showScene(2);
             }
 
-            if (self.window._gaq) {
-                self.window._gaq.push(['_trackPageview', self.virtualUrl]);
+            if (self.window.gaTrack) {
```

Comment:
After checking with the team, this is likely another oddball file that shouldn't be modified. We may need to do more investigation if this is an important file to track, but for now let's leave it unmodified.